src/opnsense/mvc/app/models/OPNsense/Proxy/Migrations/M1_0_0.php

```diff
@@ -34,4 +34,11 @@
 
 class M1_0_0 extends BaseModelMigration
```

Comment:
@AndyX90 you need to increase the version number and add a matching migration, I recently added some documentation about the subject here https://docs.opnsense.org/development/frontend/models_migrations.html. (people already have 1.0.0 or later installed, so this migration likely won't execute)

Reply:
I'm sorry, didn't understand that before. I hope this is okay now?

Reply:
@AndyX90 no problem at all, the documentation wasn't in place until this week as well.